Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/Reflection/LegacyReflectionBridge.cpp

```diff
@@ -571,13 +574,13 @@ namespace AZ::Reflection
                         return;
                     }
 
-                    AZ::Name name = propertyEditorSystem->LookupNameFromId(it->first);
+                    Name name = propertyEditorSystem->LookupNameFromId(it->first);
                     if (!name.IsEmpty())
                     {
-                        // If an attribute of the same name is already loaded then ignore the new value
-                        // unless it is a GenericValue attribute since each represents an individual
-                        // pair destined for a combobox and thus multiple are expected
-                        if (visitedAttributes.find(name) != visitedAttributes.end() && name != genericValueName)
+                        // If an attribute of the same name is already loaded then ignore the new value unless
+                        // it is a GenericValue or EnumValueKey attribute since each represents an individual
+                        // (value, description) pair destined for a combobox and thus multiple are expected
+                        if (visitedAttributes.find(name) != visitedAttributes.end() && name != genericValueName && name != enumValueKeyName)
```

Comment:
`visitedAttributes.contains(name)` can be used here.

Reply:
Embarrassed that I knew that and didn't notice it. Fixing.